charter.md

```diff
@@ -0,0 +1,51 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+## Governance
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established.
```

Comment:
It's not defined whether consensus-breaking vetoes will be recorded in public notes or not. Unless we can agree to make this process fully public, let's be explicit that it's not. For example:

```suggestion
The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition. Decisions can be made both in the repository and in meetings, as long as consensus can be established. This process will be member-confidential; vetoes preventing consensus are not required to be made public.
```

Same goes for the votes for chair selection, and induction of new members.

Reply:
I would also be OK with a simpler "everything is public" and no mention of member-confidential decisions in the charter.

Reply:
I suggest we note that the charter does not _require_ us to publish individual positions when forming consensus.

Reply:
I think the distinction is whether the charter allows for _any_ decisions to be member-confidential, or if it allows for _some_ decisions to be member-confidential under certain circumstances. I would interpret "individual positions when forming consensus are not required to be made public" as the former. Is that the intention? Is there a downside to being public by default and requiring us to document where we allow for member-confidential decisions ahead of time? Clarity ahead of time is what we were missing in the Interop 2023 process, and what I would like to have.

Reply:
It's **not** about public vs member confidential. 
It's about diverse  opinions vs one voice. It's what we need to keep in mind in that discussion. 

What are the goals of interop and how do we best achieve these goals? The goal is to bring together everyone and show unity in an effort to remove implementation differences for web developers. 

The diversity of opinions in Web standards work is absolutely necessary and hence should be recorded in public, not as a tool for positions, but as a tool for diversity of technical opinions and how an idea grow into something, how the ideas evolve. 

I don't have the feeling this is the case for Interop work. This is a restricted group of organizations which are trying to define a series of actions to improve interoperability and make a better Web. It's a medical team around a sick body with one treatment.

Reply:
@karlcow do you think the charter should say anything about how decisions are recorded? My original intention was to not mention it and handle it in the Interop 2024 process document, but since @dandclark raised the issue I think it's just as well we document it for clarity.

It seems reasonable to me to say that everything is public except where we've documented otherwise ahead of time, but if not, can someone suggest alternative wording?

Reply:
Let me come back to you a in a couple of weeks. Busy pipe this week.

Reply:
I honestly don't think the charter needs to say anything here, and to the extent it does it should note that it's not imposing any specific requirements. I think the Interop 2024 documentation should explicitly note that the outcomes of proposal selection will be communicated on behalf of the Interop group as a whole, since I think that led to good outcomes last year.

Reply:
I would be OK with saying nothing here and sorting it out in the Interop 2024 process. I hope we can all agree that if the charter doesn't say, then it neither requires or rules out member-confidential decision making.

Reply:
@karlcow @gsnedders would saying nothing here be OK with you?

Reply:
Per https://github.com/web-platform-tests/interop/pull/102#issuecomment-1623985431 it's clear that MS consider this still unresolved.

Unclear why this got resolved by @nairnandu?

Reply:
@dandclark said at the meeting last week that he is ok with saying nothing for now, in the interest of landing a consensus document. That is why Nandu closed the comment.

Reply:
I suggested that as a possible outcome given lack of progress here but not quite ready to commit it yet, still need to confirm with some folks. Would much prefer to have a real resolution on this.